html/styles.css

```diff
@@ -0,0 +1,477 @@
+}
+
+.page-header .checkout-balance .dollar {
+  font-weight: 600;
+  font-size: 8pt;
+}
+
+.page-header .main-title {
+  margin-left: 100px;
+  margin-top: 0;
+  font-family: sans-serif;
+  font-size: 14pt;
+  color: #81B841;
+}
+
+.page-header .button-checkout {
+  background-color: #81B841;
+  border: none;
+  color: white;
+  padding: 5px 20px;
+  font-size: 7pt;
+}
+
+.page-header .button-checkout {
+  position: absolute;
+  right: 95px;
+  top: 50px;
+  text-transform: capitalize;
+  text-decoration: none; 
+}
+
+.page-header .button-checkout:hover {
+  opacity: 0.9;
+}
+
+.topnav {
+  background-color: #e0e0e0;
+  padding: 20px 10%;
+  font-size: 10pt;
+  
+  font-weight: bold;
+  word-spacing: 5px;
+}
+
+.navigation-link {
+  text-decoration: none;
+  color: #134350;
+  font-weight: 800;
+}
+
+main {
+  width: 80%;
+  margin: auto;
+  padding-bottom: 50px;
+}
+
+.home-main {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.home-section {
+  flex-shrink: 2;
+}
+
+.blog-preview {
+  flex-basis: 450px;
+  display: none;
+  padding: 0 4rem;
+ 
+}
+
+
+@media only screen and (min-width: 1200px) {
+  .blog-preview {
+    display: block;
+  }
+}
+
+.blog-preview .title {
+  color: #134350;
+  font-size: 1.8rem;
+  text-transform: capitalize;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.blog-preview-snippet {
+  margin-bottom: 4rem;
+}
+
+.blog-preview-snippet .picture {
+  display: block;
+  width: 100%;
+}
+
+.blog-preview-snippet .subtitle {
+  color: #134350;
+  font-size: 1.2rem;
+  margin: 1rem 0 0 0;
+  text-transform: capitalize;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.blog-preview-snippet .summary {
+  font-size: 0.9rem;
+  line-height: 1.4rem;
+}
+
+.blog-preview-snippet .read-more {
+  color: #8181b6;
+  font-size: 0.9rem;
+  text-decoration: none;
+}
+
+.blog-preview-snippet .read-more:hover {
+  text-decoration: underline;
+}
+.blog-preview-snippet .read-more::after {
+  content: " >>";
+}
+
+.introduction {
+  display: flex;
+  align-items: flex-start;
+  margin: 20px 0;
+}
+
+.introduction .intro-image {
+  width: 35%;
+  margin-right: 40px;
+}
+
+.introduction .intro-text .title {
+  color: #134350;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.introduction .intro-text .intro-paragraph {
+  font-size: 12pt;
+  color: #444;
+  line-height: 25px;
+}
+
+@media only screen and (max-width:500px) {
+  .introduction {
+    flex-direction: column;
+    align-items: flex-start;
+  
+  }
+
+  .introduction .intro-image {
+    width: 100%;
+    margin-right: 40px;
+    display: block;
+  }
+}
+
+.dog-grid, .all-dogs-grid {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 20px;
+  justify-content: center;
+  margin-bottom: 30px;
+}
+
+.all-dogs-grid {
+  grid-template-columns: auto auto auto auto;
+}
+
+@media only screen and (max-width: 768px) {
+  .dog-grid {
+    grid-template-columns: auto auto; 
+  }
+
+  .all-dogs-grid {
+    grid-template-columns: auto auto auto;
```

Comment:
You will want the same number of columns for the home page and the dog page

Reply:
I thought the dog page is suppose to have 4 columns when I looked at the mockup. This is why I added a second class name to the home page grid so they behave differently.